tests/app/billing/test_billing.py

```diff
@@ -282,118 +284,90 @@ def test_create_free_sms_fragment_limit(client, sample_service):
     json_resp = json.loads(response_get.get_data(as_text=True))
     assert response.status_code == 201
     assert response_get.status_code == 200
-    assert json_resp['data']['financial_year_start'] == 2017
-    assert json_resp['data']['free_sms_fragment_limit'] == 250
+    assert json_resp['financial_year_start'] == get_current_financial_year_start_year()
```

Comment:
Will this still work for future years? as you are doing a `client.get` on 2017? 
Also it's probably better to use a db session to check the data rather than the request GET